testing/beacon-api/tests/serialization.rs

```diff
@@ -0,0 +1,67 @@
+use std::{fs, io::Read};
+
+use alloy_primitives::b256;
+use ream_consensus::deneb::{beacon_block::SignedBeaconBlock, beacon_state::BeaconState};
+use ream_rpc::types::response::BeaconVersionedResponse;
+use serde_json::Value;
+
+#[tokio::test]
+async fn test_beacon_state_serialization() -> anyhow::Result<()> {
+    pub type Response = BeaconVersionedResponse<BeaconState>;
+
+    let file_path = "./tests/assets/state.json";
+    let original_json = read_json_file(file_path)?;
+    println!("Serialization initiated");
+
+    let beacon_state: Response = serde_json::from_value(original_json.clone())?;
+
+    assert_eq!(beacon_state.version, "deneb");
+    assert_eq!(beacon_state.data.latest_block_header.slot, 1);
+    assert_eq!(
+        beacon_state.data.latest_block_header.parent_root,
+        b256!("0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2")
+    );
+
+    let serialized_json: Value = serde_json::to_value(&beacon_state)?;
+
+    assert_eq!(
+        original_json, serialized_json,
+        "Original JSON and re-serialized JSON do not match!"
+    );
+
+    println!("State Serialization completed successfully");
+
+    Ok(())
+}
+#[tokio::test]
+async fn test_beacon_block_serialization() -> anyhow::Result<()> {
+    pub type Response = BeaconVersionedResponse<SignedBeaconBlock>;
+
+    let file_path = "./tests/assets/block.json";
+    let original_json = read_json_file(file_path)?;
```

Comment:
```suggestion
    let original_json = read_json_file("block.json")?;
```

Can `./tests/assets/` be a const, and you pass in the extended file name + path